frontend/src/store/FamLoginUserState.ts

```diff
@@ -28,10 +31,10 @@ export interface FamLoginUser {
 }
 
 interface IMyPermission {
-    application: string,
-    env: string,
-    clientId: number,
-    role: string,
+    application: string | null | undefined;
+    env: AppEnv | null | undefined;
```

Comment:
Maybe I misunderstand, why do they need "null | undefined"? I thought these two are necessary?

Reply:
it will fail build without "**null | undefined**"

Reply:
Seems like it's because the `FamGrantDetailDto` has application description and env type "string | null", and we need to match the type here as well, otherwise it will fail
<img width="624" alt="Screen Shot 2024-05-10 at 9 48 48 AM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/64283a52-e15c-42ef-bcbe-56b402aaea71">

Reply:
I see, looks like its the backend schema. Thanks Catherine!